src/core/unit.c

```diff
@@ -5534,9 +5534,14 @@ int unit_pid_attachable(Unit *u, pid_t pid, sd_bus_error *error) {
 }
 
 void unit_log_success(Unit *u) {
+        int level;
+
         assert(u);
 
-        log_unit_struct(u, LOG_INFO,
+        /* Let's log this message for users on DEBUG level. For PID1 log on INFO level as usual. */
```

Comment:
Please update the comment to say _why_ the level is being downgraded, rather than merely stating the fact that it is, as that's obvious.

While you are there, the check is indented simple enough that it would be readable and shorter inlined without the extra variable, eg:

`log_unit_struct(u, MANAGER_IS_USER(u->manager) ? LOG_DEBUG : LOG_INFO,`

Reply:
@bluca Thank you for your review, I updated the comment and got rid of the extra variable.